app/app.go

```diff
@@ -769,6 +773,15 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(app.BaseApp.GRPCQueryRouter(), clientCtx, app.interfaceRegistry)
 }
 
+// RegisterUpgradeHandlers returns upgrade handlers
```

Comment:
Is this code needed?

Reply:
Registering upgrade handler is essential here.

Reply:
I think hes referring to it being commented out?

Reply:
Oh Yes that part I'll remove it since we move the handler to upgrade.go